controllers/tab/issueEntry/form/IssueEntryPublicationMetadataForm.inc.php

```diff
@@ -234,10 +234,17 @@ function execute($request) {
 
 			$user = $request->getUser();
 
+			// Get a list of author user IDs
+			$authorUserIds = array();
+			$stageAssignmentDao = DAORegistry::getDAO('StageAssignmentDAO');
+			$submitterAssignments = $stageAssignmentDao->getBySubmissionAndRoleId($submission->getId(), ROLE_ID_AUTHOR);
+			$submitterAssignment = $submitterAssignments->next();
```

Comment:
This might create a payment object related to a translator that has nothing to do with the article other than translating it... Why not get the principal contact of the article? I think we have that for OJS also right?